coremltools/converters/_converters_entry.py

```diff
@@ -66,111 +69,142 @@ def convert(
 
         PyTorch:
             - ``outputs`` must not be specified.
 
     classifier_config: ClassifierConfig class (optional)
-        The configuration if the mlmodel is intended to be a classifier.
+        The configuration if the MLModel is intended to be a classifier.
 
     minimum_deployment_target: coremltools.target enumeration (optional)
-        - one of the members of enum ``coremltools.target``.
-        - The value of this parameter determines the type of the model reperesentation
-          produced by the converter. Alternatively, the ``convert_to`` argument can be used to
-          specify the model type as well (see description of ``convert_to`` argument).
-        - if minimum_deployment_target <= coremltools.target.iOS14/
+        - One of the members of enum ``coremltools.target``.
+        - The value of this parameter determines the type of the model
+          reperesentation produced by the converter. Alternatively, you can use
+          the ``convert_to`` parameter to specify the model type (see the
+          ``convert_to`` parameter). To learn about the differences between
+          neural networks and ML programs, see `ML Programs <https://coremltools.readme.io/docs/ml-programs>`_.
+        - The converter produces a neural network (``neuralnetwork``) if:
+          ::
+             minimum_deployment_target <= coremltools.target.iOS14/
                                           coremltools.target.macOS11/
                                           coremltools.target.watchOS7/
                                           coremltools.target.tvOS14:
-            - a model of type neural network is produced.
-        -  if minimum_deployment_target >= coremltools.target.iOS15/
+        - The converter produces an ML program (``mlprogram``) if:
+          ::
+             minimum_deployment_target >= coremltools.target.iOS15/
                                            coremltools.target.macOS12/
                                            coremltools.target.watchOS8/
                                            coremltools.target.tvOS15:
-            - a model of type ML Program is produced.
-        - if this argument is specified and ``convert_to`` is also specified then they must be compatible.
-            Invalid values are (convert_to="neuralnetwork", minimum_deployment_target=coremltools.target.iOS15)
-            or (convert_to="mlprogram", minimum_deployment_target=coremltools.target.iOS14) etc.
-        - When both ``minimum_deployment_target`` and the ``convert_to`` arguments are unspecified,
-          converter aims to produce the neural network model type with as minimum of a deployment target as possible.
+        - If this parameter is specified and ``convert_to`` is also specified,
+          they must be compatible. The following are examples of invalid values:
+          ::
+             convert_to="neuralnetwork",
+                          minimum_deployment_target=coremltools.target.iOS15
```

Comment:
The indent in the second line is distracting and not required

Reply:
```
(convert_to="neuralnetwork", minimum_deployment_target=coremltools.target.iOS15)
```

Reply:
OK, fixed formatting errors like this one in the next commit.

Reply:
I have added a **source** link to each class/method signature, using the Sphinx extension `sphinx.ext.viewcode`. The 2nd commit includes this addition to conf.py for this extension, as well as formatting fixes for `convert`.

Reply:
See [this preview](https://tonybove-apple.github.io/coremltools/source/coremltools.converters.mil.html) and look for the tiny **source** link in the signature for each object.